interface/elements/msc-profile-view/msc-profile-view.js

```diff
@@ -215,3 +299,68 @@ Polymer({
     this.$.sender.value = "";
   }
 });
+  
+    var menu = new nw.Menu({ type: 'menubar' });
+    var account = new nw.Menu();
+    account.append(new nw.MenuItem({ label: 'New Account', key: 'n', modifiers: 'ctrl', click: function() { document.querySelector("msc-profile-view").handleNewAccount(); } }));
+    account.append(new nw.MenuItem({ label: 'Import Account', key: 'i', modifiers: 'ctrl', click: function() { document.getElementById('fileDialog').click(); } }));
+    account.append(new nw.MenuItem({ type: 'separator' }));
+    account.append(new nw.MenuItem({ label: 'Open Keystore (manual backup)', key: 'b', modifiers: 'ctrl', click: function() { document.querySelector("msc-profile-view").backupWallet(); } }));
+    account.append(new nw.MenuItem({ type: 'separator' }));
+    account.append(new nw.MenuItem({ label: 'Quit', key: 'q', modifiers: 'ctrl', click: function() { require('process').exit(0); } }));
+    menu.append(new nw.MenuItem({label: 'Account', submenu: account }));
```

Comment:
Let's add another row for sent tx list here.

Reply:
This could be not possible if you mean add all accounts here, person could have a lot accounts.
And we need update menu with every new account. With near/over 50 accounts menu will be unusable if we add it.
Or you mean another thing by `sent tx list`?

Reply:
Or I think you mean just tx window, but with custom account field.